runtime/float.go

```diff
@@ -184,6 +184,11 @@ func floatNonZero(f *Frame, o *Object) (*Object, *BaseException) {
 	return GetBool(toFloatUnsafe(o).Value() != 0).ToObject(), nil
 }
 
+func floatPow(f *Frame, v, w *Object) (*Object, *BaseException) {
+	return floatArithmeticOp(f, "__pow__", v, w, func(v, w float64) float64 { return math.Pow(v, w) })
```

Comment:
`math.Pow()` already takes 2 floats and returns another, so I think this could be reduced:

```go
func floatPow(f *Frame, v, w *Object) (*Object, *BaseException) {
    return floatArithmeticOp(f, "__pow__", v, w, math.Pow)
}
```